scene/resources/world_2d.cpp

```diff
@@ -387,7 +386,7 @@ World2D::World2D() {
 
 	//set space2D to be more friendly with pixels than meters, by adjusting some constants
 	Physics2DServer::get_singleton()->space_set_active(space, true);
-	Physics2DServer::get_singleton()->area_set_param(space, Physics2DServer::AREA_PARAM_GRAVITY, GLOBAL_DEF("physics/2d/default_gravity", 98));
+	Physics2DServer::get_singleton()->area_set_param(space, Physics2DServer::AREA_PARAM_GRAVITY, GLOBAL_DEF("physics/2d/default_gravity", 9.8));
```

Comment:
Sounds like it may not have been a good proposal from me to default to 9.8 in 2D.  See:
```
<Akien> reduz: Why does the 2D physics server use 98 as default gravity instead of 9.8?
<bojidar_bg> Akien: since 3D units are way larger than 2D pixels
   and Godot's 2D physics engine is tuned for pixels
<Akien> Ah, with 10px = 1m?
<bojidar_bg> 10px = 1u, more like
```